pkg/tasks/thanos_ruler_user_workload.go

```diff
@@ -254,7 +255,22 @@ func (t *ThanosRulerUserWorkloadTask) create() error {
 	}
 
 	err = t.client.CreateOrUpdateServiceMonitor(trsm)
-	return errors.Wrap(err, "reconciling Thanos Ruler ServiceMonitor failed")
+	if err != nil {
+		return errors.Wrap(err, "reconciling Thanos Ruler ServiceMonitor failed")
+	}
+	fmt.Println("thanos ruler")
```

Comment:
to be removed?

Reply:
No words from my side, apart from "don't push before coffee" 🤦‍♂